aks-node-controller/helpers/utils.go

```diff
@@ -5,17 +5,14 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
changed tab spacing?

Reply:
The linter did that, not me!
But I see it's all using tabs in this file and other files. So I think we should be good, right?